contracts/src/v0.8/automation/dev/interfaces/v2_3/IWrappedNative.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IERC20} from "../../../../vendor/openzeppelin-solidity/v4.8.3/contracts/token/ERC20/IERC20.sol";
+
+interface IWrappedNative is IERC20 {
```

Comment:
what contract will implement deposit and withdraw functions?

Reply:
A wrapped native token is in essence an ERC20 token with a deposit and withdraw function added ([ref](https://jamesbachini.com/wrapped-tokens/)).  Here is [WETH](https://etherscan.io/address/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2#code) and [WMatic](https://polygonscan.com/token/0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270#code).

I learnt this from [CCIP](https://github.com/smartcontractkit/ccip/blob/6190e3a2c74fb49eea00da103b1e9d681d673dc2/contracts/src/v0.8/ccip/Router.sol#L124). cc @connorwstein for any comments.